hack/bootstrap-cluster.sh

```diff
@@ -1,6 +1,37 @@
 #!/bin/bash -e
 
+# Print help message
+function print_help() {
+  echo "Usae: $0 MODE [-t|--toolchain] [-kc|--keycloak] [-h|--help]"
+  echo "  MODE             upstream/preview (default: upstream)"
+  echo "  -t, --toolchain  (only in preview mode) Install toolchain operators"
+  echo "  -kc, --keycloak  (only in preview mode) Configure the toolchain operator to use keycloak deployed on the cluster"
+  echo "  -h, --help       Show this help message and exit"
+  echo
+  echo "Example usage: \`$0 preview --toolchain --keycloak"
+}
+
 MODE=$1
```

Comment:
move MODE=$1 after while

Reply:
Sorry @Michkov, I don't understand this.
If I move that line after `while` loop, it would break the script, because of the `shift`s in that `while` will shift the positional parameters.
The way it's now it stores the `MODE` first (this is the first mandatory parameter - I could maybe figure out some validation here though) and then shifts through remaining positional parameters in `while` loop.

Reply:
right, in that case could you put it into '*)'? So that it will be possible to run `./hack/bootstrap-cluster.sh --toolchain --keycloak preview`